library/src/main/java/io/github/luizgrp/sectionedrecyclerviewadapter/SectionedRecyclerViewAdapter.java

```diff
@@ -405,6 +424,33 @@ public int getSectionPosition(String tag) {
     }
 
     /**
+     * Return the section position in the adapter.
+     *
+     * @param section This section
+     * @return position of the section in the adapter
+     */
+    public int getSectionPosition(Section section) {
```

Comment:
Thanks. I had amended accordingly. But, I'm still not sure how can create another PR just to include this method yet...